compose/src/main/java/com/bottlerocketstudios/compose/snippets/SnippetsPreviewData.kt

```diff
@@ -0,0 +1,32 @@
+package com.bottlerocketstudios.compose.snippets
+
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import com.bottlerocketstudios.compose.util.StringIdHelper
+import com.bottlerocketstudios.compose.util.asMutableState
+
+internal val mockSnippet1 = SnippetUiModel(
+    id = "123456789XYZ",
+    workspaceSlug = "workspace_slug",
+    title = "Private",
+    userName = "Bob Ross",
+    formattedLastUpdatedTime = StringIdHelper.Raw(""),
+)
+
+internal val listOfMockSnippets = listOf(mockSnippet1, mockSnippet1)
+
+class SnippetBrowserPreviewProvider : PreviewParameterProvider<SnippetsBrowserScreenState> {
```

Comment:
@BottleRocket-Colin nit: Can this class be made `internal`?

```suggestion
internal class SnippetBrowserPreviewProvider : PreviewParameterProvider<SnippetsBrowserScreenState> {
```

This is interesting!